src/Avalonia.Controls/ThemeVariantScope.cs

```diff
@@ -15,6 +15,11 @@ public class ThemeVariantScope : Decorator
         public static readonly StyledProperty<ThemeVariant?> RequestedThemeVariantProperty =
             ThemeVariant.RequestedThemeVariantProperty.AddOwner<ThemeVariantScope>();
 
+        static ThemeVariantScope()
+        {
+            RequestedThemeVariantProperty.OverrideDefaultValue<ThemeVariantScope>(ThemeVariant.Default);
```

Comment:
nit: again, it should be Default by default already. See https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Base/Styling/ThemeVariant.cs#L29-L31

Reply:
Same issue as Application